core/model/src/main/java/org/eclipse/sw360/antenna/model/artifact/facts/ArtifactClearingState.java

```diff
@@ -35,6 +36,10 @@ public String getFactContentName() {
         EXTERNAL_SOURCE,
         AUTO_EXTRACT,
         PROJECT_APPROVED,
-        OSM_APPROVED
+        OSM_APPROVED;
+
+        public static String toSafeName(@Nullable ClearingState clearingState) {
```

Comment:
Is this method still needed after the changes on the test class?

Reply:
You're right, I removed it.